webserver/templates/datasets/evaluate.html

```diff
@@ -20,6 +20,12 @@ <h2 class="page-title">Evaluate dataset "{{ dataset['name'] }}"</h2>
         <label class="col-sm-2 control-label">{{ form.filter_type.label.text }}</label>
         <div class="col-sm-4">{{ form.filter_type(class="form-control", required="required") }}</div>
       </div>
+      {% if config['EVAL_LOCATION'] %}
```

Comment:
I don't see this value in example config file. This will fail if it's not defined. Better use `config.get('EVAL_LOCATION')`.